common/src/main/java/org/jboss/hal/testsuite/creaper/command/BackupAndRestoreAttributes.java

```diff
@@ -0,0 +1,160 @@
+package org.jboss.hal.testsuite.creaper.command;
+
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.Property;
+import org.jgrapht.experimental.dag.DirectedAcyclicGraph;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Batch;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.ReadResourceOption;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * @author Jan Kasik <jkasik@redhat.com>
+ *         Created on 27.11.15.
+ */
+public class BackupAndRestoreAttributes {
```

Comment:
make the command classes `final` if you don't expect to be changed by inheritance => will make it easier when putting directly to creaper.